README.md

```diff
@@ -16,8 +39,13 @@ In this project, we hope to build a set of reference applications that make full
 
 
 
-## WE ARE JUST GETTING STARTED
-Sorry about going all caps there, but these are all in the earliest stage of development. We are looking for people to help us create these apps! 
+<hr>
+
+
+
+
+## This repo is just getting started
+All the code in this repo is  in the earliest stage of development. We are looking for people to help us create these apps! 
 
 Our goal is to show how you can perform tasks that are common in native applications using this new set of APIs. We want to keep the apps simple, so they can be easily bisected and copied by anyone.
 
```

Comment:
There's a list with a slightly larger scope curated at https://jakearchibald.github.io/isserviceworkerready/resources.html (it includes docs/libraries/articles in addition to reference demos). It would be nice to get that linked to from the end of this doc.